.travis/opensuse/Dockerfile.deps.tmpl

```diff
@@ -0,0 +1,31 @@
+FROM @IMAGE@
+
+MAINTAINER Stephen Gallagher <sgallagh@redhat.com>
+
+ARG TARBALL
+
+RUN zypper install --no-confirm --no-recommends \
+	clang \
+	clang-checker \
+	createrepo_c \
+	elinks \
+	gcc \
+	gcc-c++ \
+	git-core \
+	glib2-devel \
+	gobject-introspection-devel \
+	gtk-doc \
+	libyaml-devel \
+	meson \
+	ninja \
+	pkgconf \
+	python3-autopep8 \
+	python3-devel \
+	python3-GitPython \
+	python3-gobject \
+	python3-pycodestyle \
+	python3-rpm-macros \
```

Comment:
This package does not exist in openSUSE and appears to break the test when I try to run it locally with `./.travis/travis-opensuse.sh`

@Conan-Kudo Is there an equivalent package that should replace it?

Reply:
It's just `python-rpm-macros` in openSUSE.